### ISSUE TITLE
tigera-operator manifest and chart for Calico v3.22.2 have old CRDs

### ISSUE THREAD

**Issue by philsphicas** [projectcalico/calico#5927]
The CRDs included in the v3.22.2 tigera-operator Helm chart and tigera-operator manifest are out of sync with libcalico-go.

These changes are not reflected in the manifest CRDs:

https://github.com/projectcalico/calico/commits/release-v3.22/libcalico-go/config/crd

```
# git log --oneline 387e0c2ea..release-v3.22 libcalico-go/config/crd/
331fbdbe6 Mark sequence number as optional
3def01636 Add FelixConfiguration option for FloatingIPs (#5865)
7d83fb8db Update generated files
e232ecfa4 Add race-safe IP address release, without need for handle
9201dbb2c include CPU profiling
639bfb5b5 Add memory profiling for kube-controllers
15a452baa Run code generation and goimports
```
```
# git diff --stat 387e0c2ea..release-v3.22 libcalico-go/config/crd/
 libcalico-go/config/crd/crd.projectcalico.org_felixconfigurations.yaml           | 14 ++++++++++++++
 libcalico-go/config/crd/crd.projectcalico.org_ipamblocks.yaml                    | 38 ++++++++++++++++++++++++++++++++++++++
 libcalico-go/config/crd/crd.projectcalico.org_ippools.yaml                       |  2 +-
 libcalico-go/config/crd/crd.projectcalico.org_kubecontrollersconfigurations.yaml | 10 ++++++++++
 4 files changed, 63 insertions(+), 1 deletion(-)
```

## Expected Behavior

The CRDs in the installation manifests should be up-to-date.

https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
https://github.com/projectcalico/calico/releases/download/v3.22.2/tigera-operator-v3.22.2.tgz

## Current Behavior

Several recently-added CRD properties are missing from the manifests:
- floatingIPs
- logDebugFilenameRegex
- sequenceNumber
- sequenceNumberForAllocation
- debugProfilePort

In libcalico-go:

```
# git grep -e floatingIPs -e logDebugFilenameRegex -e sequenceNumber -e sequenceNumberForAllocation -e debugProfilePort libcalico-go/config/crd/
libcalico-go/config/crd/crd.projectcalico.org_felixconfigurations.yaml:              floatingIPs:
libcalico-go/config/crd/crd.projectcalico.org_felixconfigurations.yaml:              logDebugFilenameRegex:
libcalico-go/config/crd/crd.projectcalico.org_ipamblocks.yaml:              sequenceNumber:
libcalico-go/config/crd/crd.projectcalico.org_ipamblocks.yaml:              sequenceNumberForAllocation:
libcalico-go/config/crd/crd.projectcalico.org_kubecontrollersconfigurations.yaml:              debugProfilePort:
libcalico-go/config/crd/crd.projectcalico.org_kubecontrollersconfigurations.yaml:                  debugProfilePort:
```

In the installation manifest:
```
# wget -qO-  https://projectcalico.docs.tigera.io/archive/v3.22/manifests/tigera-operator.yaml | grep -e floatingIPs -e logDebugFilenameRegex -e sequenceNumber -e sequenceNumberForAllocation -e debugProfilePort
#
```

In the Helm chart:
```
# wget -qO- https://github.com/projectcalico/calico/releases/download/v3.22.2/tigera-operator-v3.22.2.tgz | tar -xOzf- tigera-operator/crds | grep -e floatingIPs -e logDebugFilenameRegex -e sequenceNumber -e sequenceNumberForAllocation -e debugProfilePort
#
```

## Possible Solution

Sync the directories, or cherry-pick https://github.com/projectcalico/calico/pull/5583 to release-v3.22

https://github.com/projectcalico/calico/tree/release-v3.22/libcalico-go/config/crd
https://github.com/projectcalico/calico/tree/release-v3.22/calico/_includes/charts/calico/crds/kdd

**Comment by caseydavenport** [comment]
Good catch - I've cherry-picked that PR to v3.22: https://github.com/projectcalico/calico/pull/5946

**Comment by miguelgmalpha** [comment]
@caseydavenport  I think there is a similar problem with the helm chart. This [PR](https://github.com/projectcalico/calico/pull/5649) is merged but the release artifact for [v3.22.2](https://github.com/projectcalico/calico/releases/download/v3.22.2/tigera-operator-v3.22.2.tgz) doesn't have the changes.

**Comment by caseydavenport** [comment]
@miguelgmalpha that's expected - those namespace changes weren't merged to the v3.22 release branch, and won't be available in a release until v3.23.0
